api/lib/infrastructure/utils/csv/CertificationResultsCsvValues.js

```diff
@@ -0,0 +1,131 @@
+import dayjs from 'dayjs';
+
+const I18N_VALUES_PREFIX = 'certification-results-csv.values';
+
+class CertificationResultsCsvValues {
+  static VALUES = {
+    REJECTED_AUTOMATICALLY_COMMENT: `${I18N_VALUES_PREFIX}.REJECTED_AUTOMATICALLY_COMMENT`,
+    CERTIFICATION_CANCELLED: `${I18N_VALUES_PREFIX}.CERTIFICATION_CANCELLED`,
+    CERTIFICATION_IN_ERROR: `${I18N_VALUES_PREFIX}.CERTIFICATION_IN_ERROR`,
+    CERTIFICATION_REJECTED: `${I18N_VALUES_PREFIX}.CERTIFICATION_REJECTED`,
+    CERTIFICATION_STARTED: `${I18N_VALUES_PREFIX}.CERTIFICATION_STARTED`,
+    CERTIFICATION_VALIDATED: `${I18N_VALUES_PREFIX}.CERTIFICATION_VALIDATED`,
+    COMPLEMENTARY_CERTIFICATION_CANCELLED: `${I18N_VALUES_PREFIX}.COMPLEMENTARY_CERTIFICATION_CANCELLED`,
+    COMPLEMENTARY_CERTIFICATION_NOT_DONE: `${I18N_VALUES_PREFIX}.COMPLEMENTARY_CERTIFICATION_NOT_DONE`,
+    COMPLEMENTARY_CERTIFICATION_REJECTED: `${I18N_VALUES_PREFIX}.COMPLEMENTARY_CERTIFICATION_REJECTED`,
+    COMPLEMENTARY_CERTIFICATION_VALIDATED: `${I18N_VALUES_PREFIX}.COMPLEMENTARY_CERTIFICATION_VALIDATED`,
+  };
+
+  constructor(i18n) {
+    this.translate = i18n.__;
+  }
+
+  getTranslation(headerKey, params = {}) {
+    return this.translate(headerKey, params);
```

Comment:
Pour etre moins verbeux:
```suggestion
    return this.translate(CertificationResultsCsvValues.VALUES[headerKey], params);
```

Reply:
J'ai préféré justement garder la version verbeuse, car si on passe le headerkey en string (exemple `this.getTranslation(CertificationResultsCsvValues.VALUES.CERTIFICATION_CANCELLED);` deviendrait `this.getTranslation('CERTIFICATION_CANCELLED');`, ça ne se verra pas tout de suite si la clé n'existe pas quand on est dans les méthodes appelantes.

Pour reformuler plus simplement, j'ai choisi la version verbeuse car elle mène plus facilement à où est défini l'énumération correspondante quand on est dans la fonction utilisatrice (exemple: `formatStatus`).

Qu'en penses-tu ?

Reply:
Vu ensemble :)